includes/admin/tools/import/class-give-import-donations.php

```diff
@@ -253,12 +253,12 @@ public function import_success() {
 						<?php
 						if ( $success ) {
 							echo sprintf(
```

Comment:
Use `printf()` instead of `echo sprintf()`

Reply:
Got it @Sidsector9 fixing that